cg/services/order_validation_service/workflows/microsalt/validation/field/utils.py

```diff
@@ -0,0 +1,7 @@
+from pydantic_core import ValidationError
+
+from cg.services.order_validation_service.models.errors import ValidationErrors
+
+
+def convert_errors(pydantic_errors: ValidationError) -> ValidationErrors:
```

Comment:
Not sure about this one, I think we should move the existing `convert_errors` up from the tomte module and extend it instead.